index.bs

```diff
@@ -847,16 +847,12 @@ these steps.
        1. [=Cleanup=] |transport| with |error|, |error| and true.
        1. Abort these steps.
      1. Let |session| be |transport|'s [=[[Session]]=].
+     1. Let |code| be |closeInfo|.{{WebTransportCloseInfo/closeCode}}.
      1. Let |reason| be an empty [=byte sequence=].
```

Comment:
Github won't let me make suggestions across deleted lines apparently, so doing this in two steps.
```suggestion
```